webpack/assets/javascripts/react_app/components/common/Search.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import helpers from '../../common/helpers';
+import {Form, FormGroup, ControlLabel, FormControl, Button} from 'react-bootstrap';
+
+class Search extends Component {
```

Comment:
I would prefer not introducing new component in this PR as the subject is introducing storybook.